backend/models/user.js

```diff
@@ -11,8 +11,15 @@ const userSchema = new mongoose.Schema({
     required: true,
     unique: true,
   },
+  password: {
```

Comment:
I like imposing some requirements on the password length. you need to make sure that these requirements are communicated on the frontend as well vial form validation on the register component.

Reply:
Sounds good, I might just take it out to not complicate things possibly